frameworks/Nim/prologue/prologue.dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nimlang/nim:latest
```

Comment:
Please tie this down to a specific version for reproducability/stability

Reply:
I started with `nimlang/nim:1.14` but that doesn't seems to be supported.
What's the current latest nim version that TFB supports?

Reply:
It's coming from dockerhub. We don't choose the support for these https://hub.docker.com/r/nimlang/nim/tags